src/HttpClient/Message/ResponseMediator.php

```diff
@@ -53,7 +53,18 @@ public static function getErrorMessage(ResponseInterface $response): ?string
             return null;
         }
 
-        return isset($content->message) && is_string($content->message) ? $content->message : null;
+        if (isset($content->message) && is_string($content->message)) {
+            return $content->message;
+        }
+
+        if (isset($content->errors) && is_array($content->errors)) {
```

Comment:
The exception structure I saw while working in the POST (createDiscount) method always had an array of errors.

Reply:
That related to the the twitter link I shared. I think we should include all the information in the exception.